pkg/cloud/services/networking/client.go

```diff
@@ -163,21 +171,55 @@ func (c networkClient) ListPort(opts ports.ListOptsBuilder) ([]ports.Port, error
 }
 
 func (c networkClient) CreatePort(opts ports.CreateOptsBuilder) (*ports.Port, error) {
-	return ports.Create(c.serviceClient, opts).Extract()
+	mc := metrics.NewMetricPrometheusContext("port", "create")
+	port, err := ports.Create(c.serviceClient, opts).Extract()
+	if mc.ObserveRequest(err) != nil {
+		return nil, err
+	}
+	return port, nil
 }
 
 func (c networkClient) DeletePort(id string) error {
-	return ports.Delete(c.serviceClient, id).ExtractErr()
+	mc := metrics.NewMetricPrometheusContext("port", "delete")
+	return mc.ObserveRequest(ports.Delete(c.serviceClient, id).ExtractErr())
```

Comment:
This would be a regression because we're now logging an error in prometheus on delete of an object which has already been deleted where we previously were not. I think we need to make this, and all Delete* methods in this interface, call ObserveRequestsIgnoreNotFound.

We have the same issue in all Delete* methods.